dev-env

```diff
@@ -0,0 +1,74 @@
+# This file is meant to be sourced, not executed:
+#    source ./dev-env
+
+# We're doing our best to keep it compatible with bash and zsh.
+
+if [ -z "$VIRTUAL_ENV" ]; then
+    if [ ! -d ".venv" ]; then
+        echo "Creating virtual environment at .venv"
+        python3 -m venv .venv
+    fi
+    echo "Activating virtual environment at .venv"
+    source .venv/bin/activate
+fi
+
+echo "Virtualenv is present and active !"
+
+if ! ls .venv/lib/python*/site-packages/procrastinate.egg-link 2>&1 1>/dev/null ; then
```

Comment:
Don't you want that both file descriptions 2 and 1 to go to /dev/null? In that case I guess you want to use `1>/dev/null 2>&1`.

Reply:
TIL !